charts/csm-authorization-v2.0/charts/redis/templates/redis-secret.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
   namespace: {{ include "custom.namespace" . }}
 type: kubernetes.io/basic-auth
 stringData:
-  {{ .Values.redisSecretProviderClass.redisPasswordKey }}: K@ravi123!
-  {{ .Values.redisSecretProviderClass.redisUsernameKey }}: dev
+  password: K@ravi123!
```

Comment:
Is this a default password that we advertise?

Reply:
Yes, it was introduced at the start of v2 in #419 but the user can expose their redis commander service to log into the UI